### PR TITLE
New package: qodem-1.0.1

### PR DESCRIPTION
**Pull request by drichline**
I apologize for the second pull-request; this is my first time contributing to Void

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/qodem/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+make_build_args="-f ./build/Makefile.generic"
+make_build_target="clean qodem"
+makedepends="ncurses-devel gpm-devel"
+short_desc="Qodem is a re-implementation of the Qmodem serial communication package"
```
Comment (abenson) [inline srcpkgs/qodem/template:9]: s/Qodem is a r/R/

Comment (drichline) [reply srcpkgs/qodem/template:9]: > s/Qodem is a r/R/

Sorry, I'm not sure what you mean by this

Comment (classabbyamp) [reply srcpkgs/qodem/template:9]: remove the "Qodem is a" from the line (it's written as a sed substitution)

File: srcpkgs/qodem/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+make_build_args="-f ./build/Makefile.generic"
+make_build_target="clean qodem"
```
Comment (classabbyamp) [inline srcpkgs/qodem/template:7]: the source has a configure script in it, so we can use the `gnu-configure` build style instead
```suggestion
build_style=gnu-configure
```

File: srcpkgs/qodem/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="ncurses-devel gpm-devel"
```
Comment (classabbyamp) [inline srcpkgs/qodem/template:8]:
```suggestion
makedepends="ncurses-devel gpm-devel miniupnpc-devel SDL-devel"
```
based on the output of the configure stage

File: srcpkgs/qodem/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Qodem is a re-implementation of the Qmodem serial communication package"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="CC0-1.0"
+homepage=https://qodem.sourceforge.net/
+distfiles=https://sourceforge.net/projects/qodem/files/qodem/${version}/qodem-${version}.tar.gz
+checksum=dedc73bfa73ced5c6193f1baf1ffe91f7accaad55a749240db326cebb9323359
+
+do_install() {
+	vbin bin/qodem
+	vman docs/qodem.1
```
Comment (classabbyamp) [inline srcpkgs/qodem/template:18]: now that we're using the proper build style, we shouldn't override `do_install` and we don't need to install the bin/man manually
```suggestion
post_install() {
```

File: srcpkgs/qodem/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/qodem/template:9]: Sorry, right.  It should read as "Re-implementation of the Qmodem serial communication package".

File: srcpkgs/qodem/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="ncurses-devel gpm-devel miniupnpc-devel SDL-devel"
+short_desc="Re-implementation of the Qmodem serial communication package"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="CC0-1.0"
+homepage=https://qodem.sourceforge.net/
+distfiles="${SOURCEFORGE_SITE}/qodem/qodem-${version}.tar.gz"
+checksum=dedc73bfa73ced5c6193f1baf1ffe91f7accaad55a749240db326cebb9323359
+
+post_install() {
+	vman docs/qodem.1
```
Comment (classabbyamp) [inline srcpkgs/qodem/template:15]: when I built it locally, it installed this file already

Comment (drichline) [reply srcpkgs/qodem/template:15]: In the future, I'll verify whether docs/licenses are installed by `configure` before doing it manually. Thank you again!!

Comment (classabbyamp) [reply srcpkgs/qodem/template]: what I like to do is `xbps-src install pkgname` (which runs everything up to `post_install()`, then look at what's in `masterdir/destdir/pkgname-version/` (`tree` is useful for this), then `xbps-src pkg pkgname` to finish building

alternatively, there's `xls pkgname` from the `xtools` package if you want to list files from the finished result

**Pull request merged**